gatling/src/gatling/kotlin/loadtest/AuthorsSimulation.kt

```diff
@@ -0,0 +1,28 @@
+package loadtest
+
+import io.gatling.javaapi.core.*
+import io.gatling.javaapi.http.*
+import io.gatling.javaapi.core.CoreDsl.*
+import io.gatling.javaapi.http.HttpDsl.*
+
+import java.util.concurrent.ThreadLocalRandom
+
+class AuthorsSimulation : Simulation() {
+
+    val getAuthors = repeat(10000).on(exec(
+        http("Authors")
+            .get("/authors")
+            .check(status().shouldBe(200))
+    ))
+
+    val httpProtocol =
+        http.baseUrl("http://localhost:8080")
+
+    val users = scenario("Users").exec(getAuthors)
+
+    init {
+        setUp(
+            users.injectOpen(rampUsers(10).during(0))
```

Comment:
I did some testing on my machine and realised that, with this settings, 10 users and 10000 repetitions, the CPUs are not really fully overloaded. The CPU reaches a work load of about 75%. Which means, the application can serve more requests.

I changed the settings to 1000 users and 100 repetitions. Perhaps the number of users is a little bit to much but the CPU is fully working at 100%. 

Could yo verify this as well? If you get the same results, maybe we can change the users and repetitions accordingly or, even better, make it configurable via cli arguments 😉

Reply:
make it configurable was easy:

now: ./gradlew gatlingRun -DparallelUsers=500 -DnumberOfRequests=500

I will execute the tests again

Reply:
I changed the quarkus code a bit
and added a maven wrapper for the minimal app

new test results are documented
When you achieve nearly the same results you should reply in the slack channel with new results

besides of that: I would recommend Quarkus or micronaut for enterprise applications

Reply:
Thanks Alex!